hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -5097,6 +5123,12 @@ private String reconfigureAllowListAllVolumes(String newVal) {
     return String.valueOf(allowListAllVolumes);
   }
 
+  private String reconfOzoneDirDeletingServiceInterval(String newVal) {
+    getConfiguration().set(OZONE_DIR_DELETING_SERVICE_INTERVAL, newVal);
+    getKeyManager().getDirDeletingService().setDirDeletingServiceInterval(newVal);
```

Comment:
We don't need this line, as I don't find any usage of "dirDeletingServiceInterval" in DirectoryDeletingService.java. So we can remove this line!